cmd/kubeadm/app/preflight/checks_test.go

```diff
@@ -761,10 +761,18 @@ func TestImagePullCheck(t *testing.T) {
 		CombinedOutputScript: []fakeexec.FakeAction{
 			// Test case1: pull only img3
 			func() ([]byte, []byte, error) { return nil, nil, nil },
-			// Test case 2: fail to pull image2 and image3
+			// Test case 2: fail to pull image2 and image3 (if the pull fails, it will be retried 5 times by default)
```

Comment:
please format the comment like so:
```
// Test case 2: fail to pull image2 and image3
// If the pull fails, it will be retried 5 times (see PullImageRetry in constants/constants.go)
```